docs/index.rst

```diff
@@ -107,11 +111,16 @@ Connect to CrateDB instance running on ``localhost``.
 .. code-block:: python
 
     >>> # Connect using SQLAlchemy Core.
+    >>> from pprint import pp
     >>> import sqlalchemy as sa
-    >>> engine = sa.create_engine("crate://localhost:4200", echo=True)
+    >>>
+    >>> dburi = "crate://localhost:4200"
+    >>> query = "SELECT country, mountain, coordinates, height FROM sys.summits ORDER BY country;"
+    >>>
+    >>> engine = sa.create_engine(dburi, echo=True)
     >>> with engine.connect() as connection:
-    >>>     with connection.execute(sa.text("SELECT * FROM sys.summits;")) as cursor:
-    >>>         print(cursor.fetchall())
+    ...     with connection.execute(sa.text(sql)) as result:
+    ...         pp(result.mappings().fetchall())
```

Comment:
After fixing it, thanks to your comment, this snippet works for me. What happens on your end?

Reply:
I am using `IPython` 7.31.1 and I get
```
<ipython-input-1-69990c3dd5ad> in <module>
      8 engine = sa.create_engine(dburi, echo=True)
      9 with engine.connect() as connection:
---> 10     with connection.execute(sa.text(query)) as result:
     11         pp(result.mappings().fetchall())
     12

AttributeError: __enter__
```

Reply:
Thanks. Indeed, this snippet does not work on SQLAlchemy 1.x. However, on Python 3.11, I receive better error messages:
```python
# SA13
TypeError: 'ResultProxy' object does not support the context manager protocol

# SA14
TypeError: 'LegacyCursorResult' object does not support the context manager protocol
```

Can you verify it works for when upgrading to SA20 in your environment, for example with `pip install 'sqlalchemy<2.1' --upgrade`? If that works, do you think we should add a note that it is a SA20-only snippet? I think it would add unnecessary noise, but I will consider it if you think it is important.

Reply:
> Let's clarify that this requires SA20.

Sure. Fixed with e02ee8aeb5. Thanks!